prog/test/hetzner_server.rb

```diff
@@ -85,13 +85,33 @@ def self.assemble
   label def wait_setup_host
     reap
 
-    hop_run_integration_spec if children_idle
+    if children_idle
+      # We shouldn't install specs by default
+      verify_specs(false)
```

Comment:
I find this code baffling. Perhaps use a keyword argument, and what's "to_exist" about? It appears to be some invariant check, but what's it guarding against?